### DISCUSSION TITLE
Add props to set ScrollArea default position

### DISCUSSION THREAD

**Discussion by csulit** [mantinedev/mantine#6606] [Feature requests]
I am currently developing a live chat application. Is there a way to set the scrollArea component position default to the bottom whenever I open a chat session?

Currently I'm doing this:

```
const scrollToBottom = () =>
  scrollViewport.current!.scrollTo({
    top: scrollViewport.current!.scrollHeight,
    behavior: 'smooth',
  });

useEffect(() => {
  scrollToBottom();
}, []);
```

**Comment by rtivital** [comment]
No, currently it is not supported, this feature may be added in future releases

**Reply by csulit** [reply]
> No, currently it is not supported, this feature may be added in future releases

Thank you.
